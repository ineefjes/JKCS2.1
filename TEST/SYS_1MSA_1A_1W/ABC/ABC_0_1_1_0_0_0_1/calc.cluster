7
/Users/jakub/WORKDIR/JKCS2.0/JKCSx/../TOOLS/STRUCTURES/JACOB/msa.xyz 0
/Users/jakub/WORKDIR/JKCS2.0/JKCSx/../TOOLS/STRUCTURES/JACOB/msam.xyz 1
/Users/jakub/WORKDIR/JKCS2.0/JKCSx/../TOOLS/STRUCTURES/ABC/nh3.xyz 1
/Users/jakub/WORKDIR/JKCS2.0/JKCSx/../TOOLS/STRUCTURES/ABC/nh4.xyz 0
/Users/jakub/WORKDIR/JKCS2.0/JKCSx/../TOOLS/STRUCTURES/ABC/oh.xyz 0
/Users/jakub/WORKDIR/JKCS2.0/JKCSx/../TOOLS/STRUCTURES/JACOB/aq.xyz 0
/Users/jakub/WORKDIR/JKCS2.0/JKCSx/../TOOLS/STRUCTURES/JACOB/aqp.xyz 1
* 10
