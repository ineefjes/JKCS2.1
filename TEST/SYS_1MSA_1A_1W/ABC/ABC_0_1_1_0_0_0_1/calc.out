    * * * * * * * * * * * * * * * * * * * * *
    *       __    ____    ______            *
    *      /  \  |  _ \  |  ___/  luster !  *
     *    / _  \ | |_| | | |               *
     *   / /_\ | |  _ <  | |     < @  @ >  * 
     *  / ___  | | |_| | | |___ _ _  o     * 
    *  /_/   |_| |____/  |_____ _ __ __  _  *
    *                                       *
    * * * * * * * * * * * * * * * * * * * * *

ABCluster 1.4
Specific flag:  official version
Platform:       Mac OS X
Compiled by:    junzhang@appliance-65.sunray.mediaways.net
Compiling date: Apr  2 2016 23:43:49
C++ compiler:   g++
C++ options:    -O3 -Os --std=c++0x 

Contributors:
Jun Zhang(University of Cologne): Design and coding.

For any suggestions, comments, and questions on the theory and
implementation of ABCluster, please feel free to contact:

Dr. Jun Zhang, ZhangJunQcc@gmail.com

If you used this program in your work, please cite this reference:

[1] Zhang, J.; Dolg, M.; ABCluster: The Artificial Bee Colony Algorithm for
    Cluster Global Optimization. Phys. Chem. Chem. Phys. 2015, 17, 24173-24181.
[2] Zhang, J.; Dolg, M.; Global Optimization of Clusters of Rigid Molecules by the
    Artificial Bee Colony Algorithm. Phys. Chem. Chem. Phys. 2016, 18, 3003-3010.

Program starts at Thu Aug  8 08:14:00 2019.
Command line: " /Users/jakub/WORKDIR/ABCluster/ABCluster-MacOSX/rigidmol-optimizer calc.inp "
Process ID:   71245
Run by:       jakub@student-3.vpn.helsinki.fi
Available CPU cores:    0
Read "OMP_NUM_THREADS": Unset

Random number testing (integers): 92824130...1022025188
Random number testing (fractals): 0.56197270...0.26355941

 -- Input/Output Parameters --
Input file name:      calc.inp
Cluster file name:    calc.cluster
Result file name:     calc.(cluster/xyz/gjf)
Temporary file name:  abclusterjakubstudent-3.vpn.helsinki.fi71245-OPT
# of saved LMs:       30
LMs are stored in:    calc-LM

 -- ABC Control Parameters --
Population size:                   30
Maximal generation:               100
Scout limit:                        4
Amplitude:                10.00000000

 -- Cluster Components: from [ calc.cluster ] --
 -------------------------------------------------
   Source                                       #
 -------------------------------------------------
   /Users/jakub/WORKDIR/JKCS2.0/JKCSx/../TOOLS/STRUCTURES/JACOB/msa.xyz     0
   /Users/jakub/WORKDIR/JKCS2.0/JKCSx/../TOOLS/STRUCTURES/JACOB/msam.xyz     1
   /Users/jakub/WORKDIR/JKCS2.0/JKCSx/../TOOLS/STRUCTURES/ABC/nh3.xyz     1
   /Users/jakub/WORKDIR/JKCS2.0/JKCSx/../TOOLS/STRUCTURES/ABC/nh4.xyz     0
   /Users/jakub/WORKDIR/JKCS2.0/JKCSx/../TOOLS/STRUCTURES/ABC/oh.xyz     0
   /Users/jakub/WORKDIR/JKCS2.0/JKCSx/../TOOLS/STRUCTURES/JACOB/aq.xyz     0
   /Users/jakub/WORKDIR/JKCS2.0/JKCSx/../TOOLS/STRUCTURES/JACOB/aqp.xyz     1
 -------------------------------------------------

Artificial Bee Colony Optimization Starts!
----------------------------------------------------------------------------------
 * Population: 30; Maximal cycles: 100
 * EM = Employed Bee; Trigonometric mutation
 * OL = Onlooker Bee; ABC/best/2(0.5)+ABC/current/2(0.5);
        Tournament selection from 5 nectars.
 * SC = Scout Bee; Random nectar; Limit = 4
----------------------------------------------------------------------------------
    #       Diversity            maxE            minE    byEM    byOL    bySC
----------------------------------------------------------------------------------
    1     22.84270669   -535.32833069   -613.87823278      16       4       0
    2      3.50598714   -594.33585711   -613.87823278      14       0       0
    3      3.50643214   -594.33585711   -613.87823278       8       1       0
    4      0.00703510   -613.85788882   -613.87823278       8       0       0
    5      0.00558410   -613.85788882   -613.87823278       5       0       0
    6      3.50771153   -594.33585711   -613.87823278       4       0       2
    7     71.32232804   -216.12481789   -613.87823278       4       0       8
    8     71.80206273   -250.46965895   -613.87823278       7       0      11
    9     65.92639621   -250.46965895   -613.87823278       7       0       5
   10 339814.49709611 1892445.84382284   -613.87823278       7       1       3
   11      3.50961079   -594.31832721   -613.87823278       8       0       1
   12     14.08135747   -535.42591768   -613.87823278       9       0       2
   13   2973.38191018  15955.92587579   -613.87823278       6       0       5
   14     23.00909710   -485.68916097   -613.87823278       7       0       7
   15     76.56113012   -229.64869870   -613.87823278       9       0       7
   16      0.01010099   -613.84775855   -613.87823278      10       1       3
   17     17.98677907   -537.81409101   -613.87823278       9       0       4
   18     49.40101440   -423.10307283   -613.87823278       4       0       4
   19     18.93489552   -536.51978115   -613.87823278       8       2       3
   20     64.29645674   -297.92802600   -613.87823278       8       0       6
   21     23.57665313   -483.31164280   -613.87823278       7       1       5
   22     14.09847709   -535.32833070   -613.87823278       8       2       5
   23     44.40648919   -432.91099889   -613.87823278       7       0       4
   24      3.49844459   -594.37947835   -613.87823278       9       0       2
   25     36.18458132   -412.28803482   -613.87823278       5       1       5
   26     34.27389793   -423.52267710   -613.87823278       8       1       5
   27     13.50768357   -540.51650379   -613.87823278       9       2       4
   28     74.83286444   -201.54943850   -613.87823278      10       0       4
   29     18.80000701   -536.51981079   -613.87823278       6       0       4
   30     22.05466453   -540.51650379   -613.87823278      10       1       3
   31     13.15319441   -540.59648035   -613.87823278      10       0       4
   32     37.11198618   -418.02856361   -613.87823278      10       0       5
   33     21.63482505   -493.34600989   -613.87823278      12       0       4
   34     32.23785445   -469.55038573   -613.87823278      11       0       5
   35      0.01000674   -613.84121458   -613.87823278      11       0       4
   36      0.01108493   -613.84773414   -613.87823278       9       0       6
   37      0.00958709   -613.84775855   -613.87823278       5       0       0
   38     13.15582762   -540.58557473   -613.87823278       5       0       3
   39     50.67351578   -349.38009054   -613.87823278       7       0       8
   40     18.72951325   -529.19961861   -613.87823278       6       0       6
   41     53.99391911   -319.89004718   -613.87823278       8       0       4
   42     69.47465517   -334.95880318   -613.87823278       9       0       6
   43     53.26222846   -358.05342835   -613.87823278       9       0       3
   44      0.00649407   -613.84775855   -613.87823278       5       0       2
   45     52.02775210   -324.03539154   -613.87823278       5       0       4
   46      0.01022574   -613.84121458   -613.87823278       6       0       8
   47      3.51624788   -594.28634503   -613.87823278       7       0       4
   48     66.84687855   -245.59808992   -613.87823278       6       0       7
   49     55.32643252   -305.60633690   -613.87823278       8       0       4
   50     13.88983253   -536.49672747   -613.87823278       7       0       3
   51     43.37742261   -372.34096260   -613.87823278       4       0       3
   52     44.17370577   -396.63987624   -613.87823278       6       0       7
   53     40.99896289   -396.63987624   -613.87823278       8       0       4
   54     19.21250114   -535.43400551   -613.87823278       6       0       7
   55      0.01305800   -613.84119017   -613.87823278      12       0       3
   56     45.61585244   -375.39438877   -613.87823278       8       0       5
   57     43.66799055   -370.60745144   -613.87823278       8       0       3
   58      0.00980120   -613.84121458   -613.87823278       5       0       5
   59      0.01198991   -613.84119017   -613.87823278       8       0       3
   60      3.51902183   -594.27096765   -613.87823278      10       0       4
   61     13.89164150   -536.48519571   -613.87823278       8       0       7
   62  82710.96267479 460166.26919360   -613.87823278       5       0       6
   63      8.66379481   -565.61032617   -613.87823278      10       0       2
   64      0.00693077   -613.85122719   -613.87823278      10       0       3
   65     13.16848374   -540.51650380   -613.87823278       6       0       4
   66     33.76805868   -425.75605393   -613.87823278       3       0       5
   67     16.74786935   -520.57442837   -613.87823278       7       0       5
   68     86.70255505   -195.80962540   -613.87823278       5       0       8
   69      0.01031866   -613.84119017   -613.87823278       6       0       4
   70     43.88151189   -369.41955568   -613.87823278       9       0       3
   71     13.79919195   -540.59648035   -613.87823278       5       0       5
   72     45.68353090   -359.37657195   -613.87823278       3       0       5
   73     13.88546361   -536.51978115   -613.87823278       7       0       4
   74     16.66260259   -522.44867345   -613.87823278       6       0       7
   75     13.69680894   -539.42787882   -613.87823278       8       0       4
   76      0.01174237   -613.84119017   -613.87823278      11       0       6
   77     13.49435624   -540.61886411   -613.87823278      11       0       3
   78     66.77991318   -241.85453652   -613.87823278       5       0       3
   79     57.84504658   -314.16711445   -613.87823278       5       0       4
   80     13.49342038   -540.61889327   -613.87823278       5       0       4
   81     27.62736016   -460.53943940   -613.87823278       2       0       6
   82     78.27625774   -177.80396664   -613.87823278       5       0       7
   83     59.61298514   -368.00978893   -613.87823278       6       0       6
   84     54.55421945   -309.95811786   -613.87823278      10       1       2
   85     61.90662657   -319.50907174   -613.87823278       9       0       6
   86     13.88716710   -536.50822769   -613.87823278       3       0       1
   87     14.20262789   -536.52255365   -613.87823278       6       0       7
   88     19.44298389   -536.48522534   -613.87823278       9       1       5
   89     66.63659379   -307.48730347   -613.87823278       9       0       3
   90      0.00863951   -613.85122719   -613.87823278       9       0       6
   91     31.38129485   -478.11836671   -613.87823278       6       0       5
   92      0.00710144   -613.85708067   -613.87823278      10       0       1
   93     66.63075667   -249.18397334   -613.87823278       5       0       5
   94     60.66897774   -275.89429335   -613.87823278       6       0       4
   95     44.45544550   -402.39708383   -613.87823278      10       0       6
   96     19.99133588   -528.41909660   -613.87823278       5       0       6
   97     13.16697493   -540.51653295   -613.87823278       9       0       6
   98     38.13942069   -412.32972974   -613.87823278      13       0       2
   99     39.66102380   -404.23715179   -613.87823278      11       0       3
  100     47.92854231   -346.87376959   -613.87823278       8       0       2
----------------------------------------------------------------------------------
Artificial Bee Colony Optimization Finished!

 -- Results Report --
 * 30 LMs will be saved in [ calc-LM ].
      #          Energy
      0   -613.87823278
      1   -613.87820838
      2   -613.87238046
      3   -613.87235606
      4   -613.86356353
      5   -613.86353913
      6   -613.86234886
      7   -613.86232446
      8   -613.85788882
      9   -613.85786442
     10   -613.85708067
     11   -613.85705626
     12   -613.85122718
     13   -613.85120278
     14   -613.84775855
     15   -613.84773414
     16   -613.84121458
     17   -613.84119017
     18   -604.86043002
     19   -594.37948060
     20   -594.36194546
     21   -594.35126377
     22   -594.33585936
     23   -594.33365656
     24   -594.31832720
     25   -594.31457058
     26   -594.28634728
     27   -594.27096765
     28   -565.61032617
     29   -551.44740370

 * Final Global Minimal Energy:   -613.87823278
 * The Global Minimum is saved as: [ calc-OPT.(cluster/xyz/gjf) ]

===== (^ _ ^) =====
Normal termination at Thu Aug  8 08:14:06 2019.
Total run time: 6 seconds = 0 days 0 hours 0 minutes 6 seconds.
